translate/rus.json

```diff
@@ -264,8 +264,8 @@
 "play as":"играть как",
 "play every":"играть каждый",
 "player":"игрок",
-"Please consider whitelisting [TCEC_URL] in your ad blocker --- or send a donation.":"Пожалуйста, рассмотрите белый список [TCEC_URL] в вашем блокировщике рекламы --- или отправьте пожертвование.",
-"points":"очки",
+"Please consider whitelisting [TCEC_URL] in your ad blocker --- or send a donation.":"Пожалуйста, добавьте в белый список в вашем блокировщике рекламы наш сайт [TCEC_URL]  --- или отправьте пожертвование.",
+"Points[DE]":"Очки[СЛВ]",
```

Comment:
Line 267: rephrase of the statement so it can be better understood by people.
Line 268: new update according to new GUI features. [СЛВ] abbreviation is referenced later, line 300-302.